redisinsight/ui/src/components/query/query-card/QueryCard.tsx

```diff
@@ -155,7 +155,7 @@ const QueryCard = (props: Props) => {
   const isSizeLimitExceededResponse = (result: Maybe<CommandExecutionResult[]>) => {
     const resultObj = result?.[0]
     // response.includes - to be backward compatible with responses which don't include sizeLimitExceeded flag
-    return resultObj?.sizeLimitExceeded === true || resultObj?.response?.includes('Results have been deleted')
+    return resultObj?.sizeLimitExceeded === true || resultObj?.response?.includes?.('Results have been deleted')
```

Comment:
Why don't we add test for it?

Reply:
it was one, added one more

Reply:
yeah, I saw it, but this one tests the optional `includes()`